simd/src/Kokkos_SIMD_Common.hpp

```diff
@@ -134,7 +138,7 @@ template <class T>
 
 template <class T, class U, class Abi,
           std::enable_if_t<std::is_arithmetic_v<U>, bool> = false>
-[[nodiscard]] KOKKOS_FORCEINLINE_FUNCTION auto operator+(
+[[nodiscard]] KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION auto operator+(
```

Comment:
Why are this and the one below now a host-only function? I guess we have a different overload for device-compatible types?